UI/DisplayLayoutScreen.cpp

```diff
@@ -68,50 +71,54 @@ bool DisplayLayoutScreen::touch(const TouchInput &touch) {
 	using namespace UI;
 
 	int mode = mode_->GetSelection();
-	if (g_Config.iSmallDisplayZoom == 0) { mode = -1; }
+	if (g_Config.iSmallDisplayZoomType == 2) { mode = -1; }
 
 	const Bounds &screen_bounds = screenManager()->getUIContext()->GetBounds();
-
 	if ((touch.flags & TOUCH_MOVE) && picked_ != 0) {
+		int touchX = touch.x - offsetTouchX;
+		int touchY = touch.y - offsetTouchY;
 		if (mode == 0) {
 			const Bounds &bounds = picked_->GetBounds();
 
 			int mintouchX = screen_bounds.w / 4;
```

Comment:
Aaahh I see now, I looked at the changed code instead of the line you mentioned ~ attention colors heh;p. Will rename that.